### PR TITLE
Updated document product information and new admin center

### LINKED ISSUE #6074: List of Products referred to is out of date
The page refers to various products as follows:

    Office 365 Midsize Business
    Office 365 Enterprise E1, E3, E4, E5, or K1
    Office 365 Education A2, A3, or A4
    Office 365 Government G1, G3, G4, or K1

but these don't correspond to the currently available products.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 94b3a5c8-08fe-a0c9-c4df-9daaab5a649e
* Version Independent ID: c7e977b2-f2b5-685f-6e84-ce243305cf77
* Content: [Create a developer site on an existing Office 365 subscription](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription)
* Content Source: [docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md)
* Product: **sharepoint**
* GitHub Login: @spdevdocs
* Microsoft Alias: **spdevdocs**

### PR DESCRIPTION
**Pull request by NeverHere**
## Category

- [x ] Content fix

## Related issues

- fixes #6074 

## What's in this Pull Request?
Updated document with new SharePoint admin center and steps. Updated product availability.

### CODE REVIEW

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ An Office 365 developer site makes it easier to get set up and start creating, t
...
-- Office 365 Enterprise E1, E3, E4, E5, or K1
-- Office 365 Education A2, A3, or A4
-- Office 365 Government G1, G3, G4, or K1
+- Microsoft 365 Business Standard
+- Microsoft 365 E3, E5
+- Microsoft 365 A3, A5
+- Microsoft 365 Government G3, G5
 
-**If you have an Office 365 Small Business subscription,** it supports only a single site collection, so you can't create a developer site collection. If you would like to learn more about Office 365 plans for your business, see [SharePoint Online: software boundaries and limits](https://office.microsoft.com/office365-sharepoint-online-enterprise-help/sharepoint-online-software-boundaries-and-limits-HA102694293.aspx).
+If you would like to learn more about Microsoft 365 plans for your business, see [SharePoint Online: software boundaries and limits](https://docs.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits).
```
Comment (andrewconnell) [inline docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md:23]: Remove locale from URL and let server redirect based on user's locale.

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md [hunk 2]
```diff
@@ -13,16 +13,16 @@ An Office 365 developer site makes it easier to get set up and start creating, t
...
     
-For more information about the Office 365 Enterprise offerings, see [Plans &amp; Pricing](https://products.office.com/business/office-365-enterprise-e1-business-software).
+For more information about the Microsoft 365 Enterprise offerings, see [Plans &amp; Pricing](https://www.microsoft.com/en-us/microsoft-365/compare-microsoft-365-enterprise-plans).
```
Comment (andrewconnell) [inline docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md:25]: Remove locale from URL and let server redirect based on user's locale.

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md [hunk 3]
```diff
@@ -40,24 +40,28 @@ For more information about the Office 365 Enterprise offerings, see [Plans &amp;
 
 4. In the navigation tree on the left, expand **Admin**, and then select **SharePoint**.
 
-5. In the **SharePoint admin center**, on the **Site Collections** tab, select **New** > **Private Site Collection**.
+5. In the **SharePoint admin center**, on the **Sites** menu, select **Active Sites** .
 
-   ![SharePoint Admin Center new site collection option](../images/SPAdminCenter_newSiteCollection.png)
+   ![SharePoint Admin Center Active Sites](/images/Create-a-developer-site-on-an-existing-Office-365-subscription/SharePoint-Admin-Center-Active-Sites.png)
```
Comment (andrewconnell) [inline docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md:45]: Don't put this image in a new folder at the root of the repo... please put it in the same location where the image you're replacing is.

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md [hunk 4]
```diff
@@ -40,24 +40,28 @@ For more information about the Office 365 Enterprise offerings, see [Plans &amp;
 
 4. In the navigation tree on the left, expand **Admin**, and then select **SharePoint**.
 
-5. In the **SharePoint admin center**, on the **Site Collections** tab, select **New** > **Private Site Collection**.
+5. In the **SharePoint admin center**, on the **Sites** menu, select **Active Sites** .
 
-   ![SharePoint Admin Center new site collection option](../images/SPAdminCenter_newSiteCollection.png)
```
Comment (andrewconnell) [inline docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md:45]: Please verify if this image is used anywhere else. If not, please remove it as part of this PR

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md (hunk 2 above)
Comment (NeverHere) [reply docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md]: Removed locale

File: docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md (hunk 3 above)
Comment (NeverHere) [reply docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md]: Adjusted to correct location

**Comment by NeverHere** [comment]
@andrewconnell  I think have made the correct updates to this pull request. Please let me know if anything else needs to be adjusted.

**Pull request merged**